### PR TITLE
Add away users functionality shift management

### PR DESCRIPTION
**Pull request by tomneeman151293**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/40799

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Comment by tomneeman151293** [comment]
Validations are failing on missing TPB for other scripts
for the script I added, GetAwayUsers, There is no way for me to update Away status automatically, so it cannot be tested as well.

see
https://github.com/demisto/etc/issues/44224

**Review by dantavori — CHANGES_REQUESTED** [review]
very nice, please check it also on a 6.2 server to make sure nothing is broken.
regarding the validation errors, got it, we will force merge this PR

**Review by bakatzir — COMMENTED** [review]
I would love to see a short demo of this, if possible. this should not affect/delay the review and merge process.

File: Packs/ShiftManagement/Scripts/AssignToNextShiftOOO/AssignToNextShiftOOO.yml [hunk 1]
```diff
@@ -34,6 +34,7 @@ timeout: '0'
 type: python
 dockerimage: demisto/python3:3.9.8.24399
 runonce: false
+runas: DBotRole
```
[the remark this answers is not in the document]
Comment (tomneeman151293) [reply Packs/ShiftManagement/Scripts/AssignToNextShiftOOO/AssignToNextShiftOOO.yml:37]: I had a validation failure, thats why I added it. I can investigate further if you want

File: Packs/ShiftManagement/Scripts/GetUsersOnCall/GetUsersOnCall_test.py [hunk 2]
```diff
@@ -38,12 +38,16 @@
 
 GetUsersOOO_RESULT = [{'Type': 1, 'EntryContext': {'ShiftManagment.OOOUsers': [{'username': 'brucewaine'}]}}]
 
+AWAY_USERS = [{'Type': 1, 'Contents': [{'username': 'batman'}]}]
```
[the remark this answers is not in the document]
Comment (tomneeman151293) [reply Packs/ShiftManagement/Scripts/GetUsersOnCall/GetUsersOnCall_test.py:41]: Fixed

**Comment by tomneeman151293** [comment]
@dantavori Anything needed here from my side apart from @bakatzir quick demo later on?

**Review by dantavori — APPROVED** [review]

File: Packs/ShiftManagement/Scripts/GetAwayUsers/README.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+---
+
+| **Path** | **Description** | **Type** |
+| --- | --- | --- |
+| CortexXSOAR.AwayUsers.id | Away user ID. | String |
+| CortexXSOAR.AwayUsers.username | Away user username. | String |
+| CortexXSOAR.AwayUsers.name | Away user name. | String |
+| CortexXSOAR.AwayUsers.phone | Away user phone. | String |
+| CortexXSOAR.AwayUsers.roles | Away user roles. | Unknown |
+| CortexXSOAR.AwayUsers.email | Away user email. | Unknown |
```
Comment (dantavori) [inline Packs/ShiftManagement/Scripts/GetAwayUsers/README.md:26]: please regenerate readme

**Comment by tomneeman151293** [comment]
Had a short demo with @bakatzir , Approve whenever you can :)

**Review by bakatzir — APPROVED** [review]
Please review briefly the widgets using the changed scripts(In the `ShiftManagement` pack) and make sure they look OK after the changes, before merging.

**Comment by tomneeman151293** [comment]
@Itay4 
Can we force merge this?
failed validation
Packs/ShiftManagement/Scripts/AssignAnalystToIncidentOOO/AssignAnalystToIncidentOOO.yml - [CJ105]
Due to a small addition i made to this script.

**Comment by Itay4** [comment]
> @Itay4 Can we force merge this? failed validation Packs/ShiftManagement/Scripts/AssignAnalystToIncidentOOO/AssignAnalystToIncidentOOO.yml - [CJ105] Due to a small addition i made to this script.

as discussed, lets add unit test please

**Comment by tomneeman151293** [comment]
@Itay4 Unit tests added, 80% coverage

**Pull request merged**